site/zh/blog/new-in-devtools-97/index.md

```diff
@@ -24,76 +23,92 @@ draft: true
 
 <!-- The **Recorder** panel is a preview feature. Our team is still actively working on it and we are looking for your [feedback](https://goo.gle/recorder-feedback) for further enhancements. -->
+“录制”面板特性目前还处于预览阶段。我们团队正在积极努力开发该功能。我们期待您的[反馈](https://goo.gle/recorder-feedback)以便于我们做进一步改进。
 
-{% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/3EpVa15PtbhFwwszqyWF.png", alt="Recorder panel", width="800", height="540" %}
+{% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/3EpVa15PtbhFwwszqyWF.png", alt="“录制器”面板", width="800", height="540" %}
 
 {# https://chromium.googlesource.com/devtools/devtools-frontend/+/ef26abc89035075bbdb08f1b26c1b8fd942ffc04 #}
 
-Chromium issue: [1257499](https://crbug.com/1257499)
+Chromium 议题: [1257499](https://crbug.com/1257499)
 
 
 <!-- ## Refresh device list in Device Mode {: #device } -->
+## 多设备模式中更新的设备列表 {: #device }
 
 <!-- [Enabling the Device Toolbar](/docs/devtools/device-mode#viewport), more modern devices are now added in the device list. Select a device to simulate its dimensions. -->
+[启用设备工具栏](/docs/devtools/device-mode#viewport)之后可以发现，设备列表中添加了更多的现代设备。请选择其中之一来模拟该设备的尺寸。
 
-{% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/Trx5NqE9RrqpWiN24iZ0.png", alt="Refresh device list in Device Mode", width="800", height="547" %}
+{% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/Trx5NqE9RrqpWiN24iZ0.png", alt="多设备模式中更新的设备列表", width="800", height="547" %}
 
 {# https://chromium.googlesource.com/devtools/devtools-frontend/+/ede4c59ac39f8281b3e372fa2e8f162c1a2a7ea2 #}
 
-Chromium issue: [1223525](https://crbug.com/1223525)
+Chromium 议题: [1223525](https://crbug.com/1223525)
 
 
 <!-- ## Autocomplete with Edit as HTML {: #code-completion } -->
+## 以 HTML 格式修改时自动补全 {: #code-completion }
 
 <!-- The **Edit as HTML** UI now supports autocomplete and syntax highlights. In the **Elements** panel, right click on an element, and select  **Edit as HTML**. Try typing a DOM property (e.g. `id`, `aria`), the autocomplete should help you find the property name you're looking for. -->
+**以 HTML 格式修改**现在支持自动补全和语法高亮了。在“元素”面板中，右键点击一个元素并选择**以 HTML 格式修改**。试着输入一个 DOM 属性（例如 `id`、`aria`），自动补全会帮助您找到您想要的属性。
 
-{% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/yWnmpCQXpsRjWbbRQ9Pi.png", alt="Autocomplete with Edit as HTML", width="800", height="472" %}
+{% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/yWnmpCQXpsRjWbbRQ9Pi.png", alt="在 HTML 格式修改中自动补全", width="800", height="472" %}
 
 {# https://chromium.googlesource.com/devtools/devtools-frontend/+/f467de3e756f998b0e9dd222ce286cb2b7cbaca0 #}
 
-Chromium issue: [1215072](https://crbug.com/1215072)
+Chromium 议题: [1215072](https://crbug.com/1215072)
 
 
 <!-- ## Improved code debugging experience {: #debugging } -->
+## 改善的代码调试体验 {: #debugging }
 
 <!-- Column numbers are now included in the output error in the Console. Having easy access to the column number is essential for debugging especially with minified JavaScript. -->
+现在在控制台中显示的错误输出也会包含列数了。方便看到列数对代码调试至关重要，尤其是在调试压缩后的 JavaScript 代码时。
```

Comment:
现在在控制台中显示的错误输出也会包含列数。列数的显示使代码调试更加便利，尤其是在调试压缩后的 JavaScript 代码。

Reply:
I'd suggest keeping "时” at the end of "尤其是在调试压缩后的 JavaScript 代码" to make the sentence complete. Otherwise it sounds a bit like missing a preposition. Otherwise accepted.